linux/scripts/init.sh

```diff
@@ -85,12 +90,23 @@ printf 'All done!\n'
 ## TODO: ideas for other scorable steps for teams:
 
 # Simulate a git repo's history a la:
-# cd /opt/app
-# git init
-# git remote add origin FAKE
-# <do some thing to change local code>
-# git add .
-# git commit -m "WIP"
+if [[ ! -d /opt/carrot-cruncher ]] ; then
+  mkdir /opt/carrot-cruncher && cp /opt/app/* /opt/carrot-cruncher
+  cd /opt/carrot-cruncher
+  git config --global --add safe.directory /opt/carrot-cruncher
+  git config --global init.defaultBranch main
+  git init
+  git config user.name "Bugs Bunny"
+  git config user.email "bugs@bigbadbunnies.com"
+  git remote add origin git@github.com/bigbadbunnies/carrot-cruncher
+  git add .
+  git commit -m "WIP"
+  git branch release/bunnies_v1 && git checkout release/bunnies_v1
+  sed -i -e 's/printing/picking/g' -e 's/money/carrots/g' -e 's/CHA-CHING/CRUNCH/g' main.go
+  git add .
+  git commit -m "Success"
+  git checkout main
+fi
```

Comment:
I'm gonna pull this out into its own branch, so that we can get the Terraform code in as-is.

Reply:
Pulled out to `feature/add-git-scoring-step`